happi/device.py

```diff
@@ -264,8 +265,14 @@ class Device(metaclass=InfoMeta):
                        enforce=dict, default={'name': '{{name}}'})
     stand = EntryInfo('Acronym for stand, must be three alphanumeric '
                       'characters', enforce=re.compile(r'[A-Z0-9]{3}$'))
-    screen = EntryInfo('The absolute path to the main control screen',
-                       enforce=str)
+    detailed_screen = EntryInfo('The absolute path to the main control screen',
+                                enforce=str)
+    embedded_screen = EntryInfo('The absolute path to the '
```

Comment:
Is absolute path the best way to do things here?

Alternatively, should there be a `screen_root` and these become relative to that?

Reply:
I think when we decide on a convention we could have this (not just for a device, but globally). Until then I'd like to keep each entry as specific as possible.

Reply:
OK, seems reasonable.